components/kserve/resources/servicemesh/routing/istio-local-gateway.yaml

```diff
@@ -15,3 +15,9 @@ spec:
         protocol: HTTPS
       tls:
         mode: ISTIO_MUTUAL
+    - hosts:
+      - '*.svc.cluster.local'
+      port:
+        name: http
+        number: 8082
```

Comment:
With this you need to explicitly use that port, right? As the service's port 80 will point to 8081 (which is then basically istio mTLS). Seems confusing.

Do you need both mTLS + http? Or could you just go with 8080 as http as per our documentation?

Reply:
The goal is to access the isvc endpoint using the service host name (*.svc.cluster.local). According to my testing, the isvc endpoint is not accessible via mTLS from pods that are not under a mesh. 

>could you just go with 8080 as http as per our documentation?
I think I missed this one, could you please share the documentation?

Reply:
Adding HTTP is a temporary solution because Kserve hasn't created a mesh VirtualService. Consequently, it can't access the endpoint through istio-proxy. I'll address this problem by reaching out to Upstream KServe to create the mesh VirtualService.

Reply:
Docs link: https://access.redhat.com/documentation/en-us/red_hat_openshift_serverless/1.30/html/integrations/serverless-ossm-setup#serverless-ossm-setup_serverless-ossm-setup

Reply:
Sorry should have been 8081 not 8080. I think what I tried to say is, either use 8081 for mTLS or for http not add both ports. As there is only one port 80 on the K8s service, you can only point to one of those two ports here anyway (see also slack discussion).